web-registry/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -48,7 +48,7 @@ const Dashboard = (): JSX.Element => {
   const isCreditClassAdmin = useQueryIfCreditClassAdmin();
   const isProjectAdmin = useQueryIfProjectAdmin();
   const walletContext = useWallet();
-  const projectTabHidden = !isIssuer || !isProjectAdmin;
+  const projectTabHidden = !isIssuer && !isProjectAdmin;
```

Comment:
Per @S4mmyb 
>You should only see the tab if you're a project admin or a credit class issuer

Therefore, we should _hide_ it if you are not an Issuer AND not a Project Admin

Reply:
I think maybe it would have been clearer to express this condition in a positive way: `showTab = isIssuer || isAdmin`